CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 ## 1.0
 
+### 1.0.0-beta.2 - Not yet released
+
+#### Breaking changes
+
+All environment variables configuring the docker-image-cleaner has been renamed
+but have the same functionality.
+
+| Before                    | After                                    |
+| ------------------------- | ---------------------------------------- |
+| `NODE_NAME`               | `DOCKER_IMAGE_CLEANER__NODE_NAME`        |
```

Comment:
Is the double-underscore a typo? Seems like it should be `DOCKER_IMAGE_CLEANER_NODE_NAME`

Reply:
Its an opinionated style choice that I think makes sense but would be willing to adjust if you disagree.

I've seen this be used in dask and have appreciated how it made it a bit more readable to have a separation between project name and config name.

Reply:
good point, reverted!